spec/system/bookmark_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+describe "Bookmarking reports attached to a group", type: :system, js: true do
```

Comment:
🏌️ all system specs have js! 😃 

```suggestion
describe "Bookmarking reports attached to a group", type: :system do
```

Reply:
Lol this is my snippet, I will update it to remove this :)